### PR TITLE
Adding in our common .editorconfig

### PR DESCRIPTION
**Pull request by weejames**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+indent_size = 4
+
+[*.scss]
+indent_size = 4
+
+[*.feature]
+indent_size = 2
+
+[*.rb]
+indent_size = 4
```
Comment (stuartmemo) [inline .editorconfig]: Is ruby not 2?

Comment (weejames) [inline .editorconfig]: Hmm.. 2 projects I looked at (Einstein & Curriculum) have it as 4.  Genres and Guides have it as 2 `¯\_(ツ)_/¯`

**Comment by djorak** [comment]
Looks good to me.

**Comment by imjacobclark** [comment]
Looks good to me, would be nice if we could get most of our style attributes in here. Particularly like the way it supports many editors! 

:+1:

**Pull request merged**